lib/ui/viewer/gallery/component/huge_gallery_list_view.dart

```diff
@@ -8,12 +8,20 @@ import "package:photos/models/file.dart";
 import "package:photos/models/selected_files.dart";
 import "package:photos/ui/common/loading_widget.dart";
 import "package:photos/ui/huge_listview/huge_listview.dart";
-import "package:photos/ui/viewer/gallery/component/lazy_loading_gallery.dart";
+import 'package:photos/ui/viewer/gallery/component/group/lazy_group_gallery.dart';
 import "package:photos/ui/viewer/gallery/gallery.dart";
 import "package:photos/utils/local_settings.dart";
 import "package:scrollable_positioned_list/scrollable_positioned_list.dart";
 
-class GalleryListView extends StatelessWidget {
+/*
+HugeGalleryListView is a widget that displays a list of grouped/collated
+files when grouping is enabled.
+For each group, it displays a header and use LazyGroupGallery to display a
+particular group of files.
+If a group has more than 400 files, LazyGroupGallery internally divides the
+group into multiple grid views during rendering.
+ */
+class HugeGalleryListView extends StatelessWidget {
```

Comment:
I couldn't come up with better name for this..
`MultipleGroupsGalleryView` `GalleryGroupedListsView` might be better candidate